models/attribute-type/src/presets/enum-type.ts

```diff
@@ -0,0 +1,8 @@
+import { AttributeEnumType } from '@commercetools/platform-sdk';
+import { EnumModel } from '../builder';
+import { TAttributeTypeBuilder } from '../types';
+
+const enumType = (): TAttributeTypeBuilder<AttributeEnumType> =>
+  EnumModel().name('enum').values([]);
```

Comment:
I like that these "special models" are used internally for presets. We do expose them, but it's a cool pattern